codegen/Package.swift

```diff
@@ -61,6 +61,10 @@ appendLibTarget(name: "rest_json_extras", path: "\(baseDirLocal)/rest_json_extra
 appendTstTarget(name: "rest_json_extrasTests", path: "\(baseDirLocal)/rest_json_extras", dependency: "rest_json_extras")
 appendLibTarget(name: "Waiters", path: "\(baseDirLocal)/Waiters")
 appendTstTarget(name: "WaitersTests", path: "./protocol-test-codegen-local/Tests", dependency: "Waiters")
+appendLibTarget(name: "NestedAggregates", path: "\(baseDirLocal)/NestedAggregates")
+appendTstTarget(name: "NestedAggregatesTests", path: "\(baseDirLocal)/NestedAggregates", dependency: "NestedAggregates")
+appendLibTarget(name: "NestedSparseAggregates", path: "\(baseDirLocal)/NestedSparseAggregates")
+appendTstTarget(name: "NestedSparseAggregatesTests", path: "\(baseDirLocal)/NestedSparseAggregates", dependency: "NestedSparseAggregates")
```

Comment:
nit: imo it'd be great to combine these. There are many many traits so I'm not sure we need to get this granular as it could grow uncontrollably. Since they are all using the `restJson1` protocol, perhaps we add these tests to the `rest_json_extras` and `rest_json_extrasTests` targets?

Reply:
I will look at consolidating these tests with the existing `restJson` tests.

I will also look at separating the waiters tests from the `restJson` suite.

Reply:
@epau The new tests here are added to `rest-json-extras` instead of their own protocol test suites.
Waiters remains its own protocol test suite, and its Smithy definitions have been split to a separate file.